cmake/modules/extensions.cmake

```diff
@@ -1295,6 +1295,16 @@ function(zephyr_linker_sources location)
     # Create strings to be written into the file
     set (include_str "/* Sort key: \"${SORT_KEY}\" */#include \"${relpath}\"")
 
+    # Remove line from other snippet file, if already used
```

Comment:
we need to document this behavior in the description of the function here:
https://github.com/zephyrproject-rtos/zephyr/blob/ddc8fbc0ccb2dcf926b7395f847e2c7f0e605716/cmake/modules/extensions.cmake#L1155-L1161

Reply:
you are right. done.